jobs/telemetry_batch_view.py

```diff
@@ -1,33 +1,30 @@
 #!/usr/bin/env python
 
+from requests
 from os import chdir
 from os import environ
 from subprocess import call
 
 
+artifact_file = "artifact.jar"
+
+
 def call_exit_errors(command):
     print("+ {}".format(" ".join(command)))
     rc = call(command)
     if rc > 0:
         exit(rc)
 
 
-def clone_and_compile_tbv():
-    command = [
-        "git",
-        "clone",
-        environ.get("GIT_PATH", "https://github.com/mozilla/telemetry-batch-view.git"),
-        "--branch",
-        environ.get("GIT_BRANCH", "master"),
-    ]
-
-    call_exit_errors(command)
+def retrieve_jar():
+    jar_url = environ.get("ARTIFACT_URL")
 
-    print("+ cd telemetry-batch-view")
-    chdir("telemetry-batch-view")
+    if jar_url is None:
+        exit(1)
 
-    command = ["sbt", "assembly"]
-    call_exit_errors(command)
+    response = requests.get(jar_url)
```

Comment:
I'm almost offended enough by the potential jar size to say we should do streaming iteration of the request. Almost.

Reply:
You're not wrong. Let's revisit in 6 months when the jars are >1GB.

Reply:
we've tried this, right? I would not be suprised if >100MB was enough cause a problem with this, but if it works then good enough.